CLAUDE.md

```
@@ -58,19 +58,19 @@ Use the built-in `fastmcp` command-line interface for an efficient development c
 #### Interactive Debugging (`fastmcp dev`)
 For interactive testing and debugging, use the `dev` command. This runs your server with the MCP Inspector UI, which allows you to call tools and inspect responses.
 
-- **Start the dev server**: `fastmcp dev crawler_mcp/server.py --with crawl4ai --with qdrant-client --with torch`
+- **Start the dev server**: `fastmcp dev crawler_mcp/crawlers/optimized/server.py --with crawl4ai --with qdrant-client --with torch`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)